src/components/shared/footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FaInstagram, FaFacebook, FaTwitter } from 'react-icons/fa';
+// import { ThemeToggle } from '../navbar/ThemeToggle';
+const year = new Date().getFullYear();
+
+function Footer() {
+  return (
+    <div className="bg-white text-dark2 dark:bg-dark dark:text-cyan-700 dark:border-primary border-t-2 border-transparent h-auto flex flex-col sm:flex-col justify-between px-8 py-3 gap-5 sm:gap-2 max-w-full mt-auto">
+      <div className="flex flex-col sm:flex-row sm:gap-20">
+        <div className="footer__links flex  justify-center sm:justify-start">
+          <a href="/about-us" className="mr-6 hover:text-cyan-300 hover:scale-110">
+            About us
+          </a>
```

Comment:
Don't wrap meaningful links in actual `anchor tags`. That is why we have a router installed. 

Wrap these links in `NavLinksFacade` or `LinkFacade` imported from the `lib` folder.